### PR TITLE
fix(SQFormGuidedWorkflow.js): add padding and update shared component…

### PR DESCRIPTION
**Pull request by shahidmanzoorzekiexperts**
When we updated the version of scplus-shared-components in SQForm then it causes a Padding issue with SQFormGuidedWorkflow component.
https://www.loom.com/share/66e8fc9316124bc4aed2c0e3fa27a2c2

Before:
before

After:
lib

### CODE REVIEW

File: src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ import {GuidedWorkflowProps} from './PropTypes';
 
 const useStyles = makeStyles(() => {
   return {
+    root: {
+      padding: '16px !important' /* material-ui override */
```
Comment (SeanGroff) [inline src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:22]: The `!important` tag was needed in version 0.x of Material-UI but shouldn't be needed when using `makeStyles`
Can you try this without the `!important` tag please?

**Review by SeanGroff — CHANGES_REQUESTED** [review]

File: src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js (hunk 1 above)
Comment (shahidmanzoorzekiexperts) [reply src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:22]: let me try

Comment (shahidmanzoorzekiexperts) [reply src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:22]: I removed !important but not working I tried.

**Review by SeanGroff — APPROVED** [review]
Thanks @shahidmanzoorzekiexperts for your contribution!

**Pull request merged**